src/rtctools_heat_network/gas_physics_mixin.py

```diff
@@ -124,6 +128,54 @@ def __gas_node_heat_mixing_path_constraints(self, ensemble_member):
 
         return constraints
 
+    def __gas_head_loss_path_constraints(self, ensemble_member):
+        constraints = []
+        parameters = self.parameters(ensemble_member)
+
+        for pipe in self.heat_network_components.get("gas_pipe", []):
+            q = self.state(f"{pipe}.Q")
+            dh = self.state(f"{pipe}.dH")
+            r = parameters[f"{pipe}.r"]
+            # v_loss_nom = parameters[f"{cable}.nominal_voltage_loss"]
+            dh_nom = parameters[f"{pipe}.nominal_head_loss"]
+            p_length = parameters[f"{pipe}.length"]
+
+            constraint_nominal = self.variable_nominal(dh)
+
+            # TODO: tobe replaced with proper headlosses relations
+            try:
```

Comment:
same as for electricity cables would prefer if we use if/else structure here for more predictable bahaviour

Reply:
agreed, I will change it